puppet/modules/freight/manifests/init.pp

```diff
@@ -1,15 +1,8 @@
 # @summary install freight
 class freight {
-  if $facts['os']['family'] == 'Debian' {
-    apt::source { 'freight':
-      location => 'http://build.openvpn.net/debian/freight_team',
-      repos    => 'main',
-      key      => {
-        id     => '30EBF4E73CCE63EEE124DD278E6DA8B4E158C569',
-        source => 'https://swupdate.openvpn.net/repos/repo-public.gpg',
-      },
-      before   => Package['freight'],
-    }
+  class {'epel':
+    epel_testing_enabled => true,
+    before               => Package['freight'],
   }
```

Comment:
What's our policy on EPEL, should modules explicitly add `include epel` (or whatever) if they use packages from there, or can we assume it to be present?

(In the past the freight module assumed it)

Reply:
If it depends on EPEL, it should include it. Preferably like this so it can be done in all modules that use EPEL.
```puppet
include epel
Class['epel'] -> Package['freight']
```